mne/cov.py

```diff
@@ -386,66 +389,90 @@ def compute_raw_covariance(raw, tmin=None, tmax=None, tstep=0.2,
     picks : array-like of int | None (default None)
         Indices of channels to include (if None, all channels
         except bad channels are used).
+    keep_sample_mean : bool (default False)
+        If False, the average response over epochs is computed for
+        each event type and subtracted during the covariance
+        computation. This is useful if the evoked response from a
+        previous stimulus extends into the baseline period of the next.
+        Note. This option is only implemented for method='empirical'.
+        If True, the epochs will only be baseline corrected.
+
+        .. versionadded:: 0.12
+
+    method : str | list | None (default 'empirical')
+        The method used for covariance estimation.
+        See :func:`mne.compute_covariance`.
+
```

Comment:
maybe add a note that  'auto' might not be the best choice here as it can be very time-consuming to compute on full raw data.